### ISSUE TITLE
Move all this stuff to ReSpec repo as markdown

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/respec-docs#46]
The process of updating this stuff is a bit annoying. We should just create markdown documents for the docs, which are viewable on github directly from the GH repo.  It will make this whole project more maintainable if it's on place and not using HTML + a custom ReSpec profile.

**Comment by halindrome** [comment]
Yeah.... it is nice that it is at w3.org/respec though.  Is there a convenient way to achieve that?

**Comment by marcoscaceres** [comment]
Redirect 😇
